presidio-anonymizer/setup.py

```diff
@@ -14,8 +14,11 @@
 with open(path.join(this_directory, "README.MD"), encoding="utf-8") as f:
     long_description = f.read()
 
-with open(os.path.join(parent_directory, "VERSION")) as version_file:
-    __version__ = version_file.read().strip()
+try:
+    with open(os.path.join(parent_directory, "VERSION")) as version_file:
+        __version__ = version_file.read().strip()
+except:  # noqa: E722
+    __version__ = os.environ.get("VERSION", "0.0.1-alpha")
```

Comment:
same  - what if the version will change?

Reply:
@navalev this introduces a fall back in case the VERSION file is not found. In this case we would use an environment variable, and the final fall back is for a default version which is considered pre-release so it wouldn't affect PyPI